docs/best-practices/caching.md

```diff
@@ -467,7 +467,7 @@ in the cloud.
 > are currently slated for retirement on November 16th, 2016.
 > It is recommended that you migrate to Azure Redis Cache in
 > preparation for this retirement. For more information, see
-> [What is Azure Redis Cache offering and what size should I use?](/azure/redis-cache/cache-faq#what-redis-cache-offering-and-size-should-i-use)e.
+> [What is Azure Redis Cache offering and what size should I use?](https://docs.microsoft.com/en-us/azure/redis-cache/cache-faq#what-redis-cache-offering-and-size-should-i-use).
```

Comment:
Links to docs.microsoft.com should just use the segment (/azure/redis-cache/....)